apicurio-registry/deploy/crds/registry_v1alpha1_apicurioregistry_crd.yaml

```diff
@@ -0,0 +1,66 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: apicurioregistries.registry.apicurio.io
+spec:
+  group: registry.apicurio.io
+  names:
+    kind: ApicurioRegistry
+    listKind: ApicurioRegistryList
+    plural: apicurioregistries
+    singular: apicurioregistry
+  scope: Namespaced
+  subresources:
+    status: {}
+  validation:
+    openAPIV3Schema:
+      properties:
+        apiVersion:
+          description: 'APIVersion defines the versioned schema of this representation
+            of an object. Servers should convert recognized schemas to the latest
+            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#resources'
+          type: string
+        kind:
+          description: 'Kind is a string value representing the REST resource this
+            object represents. Servers may infer this from the endpoint the client
+            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds'
+          type: string
+        metadata:
+          type: object
+        spec:
+          properties:
+            image:
+              type: string
+            replicas:
+              description: 'INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
+                Important: Run "operator-sdk generate k8s" to regenerate code after
+                modifying this file Add custom validation using kubebuilder tags:
+                https://book-v1.book.kubebuilder.io/beyond_basics/generating_crd.html'
+              format: int32
+              type: integer
+          required:
+          - replicas
+          - image
```

Comment:
Docker image URL should be taken from (preferential order):
* Image field from CR. Useful for devtesting
* Env var like `APICURIO_REGISTRY_IMAGE`.  Handy when operator life cycle is managed by [Operator Life Managers](https://github.com/operator-framework/operator-lifecycle-manager)
* GO constant. Default value used if value cannot be taken from previous mechanisms.